ssca-manager/chart/values.yaml

```
@@ -194,5 +194,23 @@ affinity: {}
 java:
   memory: 512m
 
+s3:
```

```suggestion
virtualService:
    annotations: {}
s3:
```

Otherwise, it will throw an error if we do not provide this in the override.